### PR TITLE
Extreme Events -Climatic Methods Menus

### PR DESCRIPTION
**Pull request by aduut**
Extreme Events Dialog with all the parameters.

### CODE REVIEW

File: instat/dlgExtremeEvents.vb [hunk 1]
```diff
@@ -1,51 +1,79 @@
...
+        ucrBase.clsRsyntax.SetFunction("climate_obj$()")
+        ucrBase.clsRsyntax.iCallType = 0
     End Sub
 
-    Private Sub chkDrySpell_CheckedChanged(sender As Object, e As EventArgs) Handles chkThresholdValue.CheckedChanged
-        UpdateVisible()
+    Private Sub txtColumnNameExtreme_Leave(sender As Object, e As EventArgs) Handles txtColumNameExtremeEvent.Leave
+        ucrBase.clsRsyntax.AddParameter("col_names", Chr(34) & txtColumNameExtremeEvent.Text & Chr(34))
+        ucrBase.clsRsyntax.AddParameter("col_names", Chr(34) & txtColumnNameEventDay.Text & Chr(34)) ' this is not quite right the col_names is a vector that takes to parameters; is of the form col_names=c(par1,par2). Not sure of a way to proceed, could use some help
+    End Sub
```
Comment (volloholic) [inline instat/dlgExtremeEvents.vb]: This is a good way to ask for help we will look at this and get back to you.

**Comment by volloholic** [comment]
We'll look at the bits you've asked for help on and get back to you.

**Comment by aduut** [comment]
Many thanks, looking forward to hearing from you.

Cheers!

*--------------------------------------------------------------*
*DUUT Abib*

*The visual symbols  below are the Adinkra Symbols which were created by
the Akan (a group of people in Ghana). These symbols represent aphorisms or
concepts. For more about these symbols click on the following links:*

*http://en.wikipedia.org/wiki/Adinkra_symbols
<http://en.wikipedia.org/wiki/Adinkra_symbols>*
*http://www.adinkra.org/ <http://www.adinkra.org/>*


*MFRAMADAN <http://www.adinkra.org/htmls/adinkra/mfra.htm>: *"wind-resistant
house":fortitude, preparedness.


NEA ONNIM NO SUA A, OHU <http://www.adinkra.org/htmls/adinkra/neao.htm>:"he
who does not know can know from learning" :knowledge, life-long education

**Comment by volloholic** [comment]
When you're ready I'll give you a temporary solution through Skype.

**Comment by maxwellfundi** [comment]
This is redone, it can now be closed

**Pull request closed without merging**